### ISSUE TITLE
Unable to add string like sensor

### ISSUE THREAD

**Issue by ultratoto14** [rospogrigio/localtuya#135]
I wanted to use sensors to help debugging light scenes.
I added a sensor in yaml with ID 25

```yaml
  - host: 192.168.15.160
    device_id: xxxxxxxxxxxxxxxxxx
    local_key: xxxxxxxxxxxxxxxx
    friendly_name: RGBW Led Strip LSC
    protocol_version: "3.3"
    entities:
      - platform: light
        friendly_name: RGBW Led Strip LSC
        id: 20
        color_mode: 21
        brightness: 22
        color_temp: 23
        color: 24
        brightness_upper: 1000
        color_temp_min_kelvin: 2700
        color_temp_max_kelvin: 6500

      - platform: sensor
        friendly_name: RGBW Led Strip LSC - 25
        id: 25
```
Even if not set a scale factor of 1.0 is applied, preventing the ability to have String sensors.

**Comment by SmartM-ui** [comment]
> I wanted to use sensors to help debugging light scenes.
> I added a sensor in yaml with ID 25
> 
> ```yaml
>   - host: 192.168.15.160
>     device_id: xxxxxxxxxxxxxxxxxx
>     local_key: xxxxxxxxxxxxxxxx
>     friendly_name: RGBW Led Strip LSC
>     protocol_version: "3.3"
>     entities:
>       - platform: light
>         friendly_name: RGBW Led Strip LSC
>         id: 20
>         color_mode: 21
>         brightness: 22
>         color_temp: 23
>         color: 24
>         brightness_upper: 1000
>         color_temp_min_kelvin: 2700
>         color_temp_max_kelvin: 6500
> 
>       - platform: sensor
>         friendly_name: RGBW Led Strip LSC - 25
>         id: 25
> ```
> 
> Even if not set a scale factor of 1.0 is applied, preventing the ability to have String sensors.

Follow

**Comment by postlund** [comment]
We have a similar fix baked into #77 which only applies scaling if taget type is int or float:

https://github.com/rospogrigio/localtuya/blob/5043a7956ab30b7fc14eff008ee04d4b7919d686/custom_components/localtuya/sensor.py#L67

But we should be able to have both changes merged.
